src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    text={article.text}
+    date={article.date}
+  />
```

Comment:
Еще можно сделать то же самое деструктуризацией в одной строке, ведь у нас всего 3 параметра которые мы знаем заранее и они ВСЕ используются.

```suggestion
   <Article {...article} />
```